_posts/2020-06-01-newsletter.md

```diff
@@ -0,0 +1,105 @@
+
+As we helped fix a [double font loading bug on ft.com](https://github.com/Financial-Times/dotcom-page-kit/pull/826) at the same time, we don't have statistics on how much the new font format improved the performance of ft.com for our readers. But both updates together made a huge improvement:
+
+- Visually complete: 8% faster
+- DOM content loaded: 17% faster
+- Total page size: 18% smaller
+
+Big thanks to Nick Colley for making the suggestion that we include `woff2`; and for helping us deploy the updates quickly to Customer Products projects!
+
+### Origami Build Tools v10
+
+We released another major of Origami Build Tools. This release includes some significant maintenance work updating a number of outdated or deprecated dependencies. But more excitingly we've made improvements to the workflow for component maintainers too 🎉
+
+_If you contribute to Origami components or your team actively maintains one we recommend upgrading to v10. Please let us know if you have any issues or feedback._
+
+#### Simplified Interface
+
+To help engineers new to Origami components get up and running with component development we simplified the most common command. To start working on a component with Origami Build Tools it was common to run the following command:
+
+```
+obt demo --run-server --watch --verbose
+```
+
+That's a lot of options to understand at first. Instead we simplified this to:
+
+```
+obt develop
+```
+
+If you've never used Origami Build Tools before and would like to understand what this command does, [see the Origami Build Tools readme](https://github.com/Financial-Times/origami-build-tools#usage).
+
+#### Speed Improvements
+
+Origami Build Tools now compiles component code faster. It's an important metric for Origami Build Tools as it helps keep the feedback loop short between an engineer making a code change and seeing the result of that change.
+
+As an unscientific example on my work laptop: the quickest of three [o-table](https://github.com/Financial-Times/o-table) demo builds is 2.18 seconds over 4.11 seconds -- around a 53% improvement.
+
+We improved the performance of Origami Build Tools by creating two new packages:
+- [Financial-Times/scrumple](https://github.com/Financial-Times/scrumple#scrumple): a fast (and scrappy) JavaScript bundler for developing Origami components, based on a now-deleted tool called Pax.
+- [Financial-Times/sass](https://github.com/Financial-Times/sass): Prebuilt [dart-sass](https://github.com/sass/dart-sass) binaries available via NPM.
+
+## Special Thanks
+
+This month our special thanks goes to Rowan Manning! In case you missed the news, at the start of May Rowan moved from the Origami team (O&R) to Customer Products. So a big thank you to Rowan for all their work on and leading Origami over the years 😃 (And congratulations again to Jake Champion, who is our new Origami tech lead!)
```

Comment:
Let's keep it to only thanking Rowan, I don't need any congratulations in the Origami Newsletter :-)
```suggestion
This month our special thanks goes to Rowan Manning! In case you missed the news, at the start of May Rowan moved from the Origami team (O&R) to Customer Products. So a big thank you to Rowan for all their work on and leading Origami over the years 😃
```

Reply:
I thought people who haven't heard might find it useful to know who the team lead is now, and thought this was a nice way to include that info :) maybe a reword? happy to remove if you'd prefer

Reply:
That makes much more sense now :-)
I'd reword it to not have the congratulations, something like this?
```suggestion
This month our special thanks goes to Rowan Manning! In case you missed the news, at the start of May Rowan moved from the Origami team (O&R) to Customer Products. So a big thank you to Rowan for all their work on and leading Origami over the years 😃. Jake Champion is the stand-in Tech Lead, if you have questions about Origami please contact them instead of Rowan :-)
```